vizro-ai/examples/dashboard_ui/app.py

```diff
@@ -79,21 +79,18 @@
     title="Vizro-AI - create interactive charts with Plotly and Vizro",
     layout=vm.Layout(
         grid=[
-            [4, 4, 4, 4],
-            [2, 2, 1, 1],
-            [2, 2, 1, 1],
-            [3, 3, 1, 1],
-            [3, 3, 1, 1],
-            [3, 3, 1, 1],
-            *[[0, 0, 1, 1]] * 8,
+            *[[4, 4, 4, 4]] * 2,
+            *[[2, 2, 1, 1]] * 3,
+            *[[3, 3, 1, 1]] * 4,
+            *[[0, 0, 1, 1]] * 10,
```

Comment:
I am not sure how the layout worked before for bigger/smaller screensizes, but when I see reduce the screen width, I see these containers still overlapping:

![Screenshot 2024-11-15 at 11 08 36](https://github.com/user-attachments/assets/bc9ed91f-66e2-4e09-a62b-ec7bfcd4ef34)

This might have to be done by switching to a flexbox model for small screen sizes, so getting rid of the grid-layout and have the form and chart underneath each other.